__test__/data.ts

```diff
@@ -30,10 +32,10 @@ export const dummyKeywords = [
        },
        url: 'https://compressimage.io/',
        tags: [],
-       lastResult: [],
+       lastResult: lastResult,
```

Comment:
I've added an option to provide `lastResult` to pass an expected value from the test case.